test/Spec/State/For.spec.luau

```diff
@@ -54,7 +54,7 @@ return function()
 			numCalls += 1
 			local k, v = peek(inputPair).key, peek(inputPair).value
 			seen[k] = v
-			return Computed(scope, function(use)
+			return Computed(scope, function(use, scope)
 				return {key = string.upper(use(inputPair).key), value = (use(inputPair) :: any).value * 10}
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
			return Computed(scope, function(use)
```

Reply:
reverted